### ISSUE TITLE
Doesnt work

### ISSUE THREAD

**Issue by kr1st4ps** [kevinatown/MMM-Screencast#30]
When I put it in the config.js and run MagicMirror, the whole MagicMirror is just black and no other modules show up either.

**Comment by kevinatown** [comment]
Hey @kr1st4ps sorry that it’s not working for ya. Can you paste your MM’s config in here (please remove all keys first :)

Thanks,
Kevin

**Comment by kr1st4ps** [comment]
[config.js.txt](https://github.com/kevinatown/MMM-Screencast/files/3254276/config.js.txt)

**Comment by kevinatown** [comment]
So nothing looks out of the ordinary with that config (besides the commented out config). When you have the MMM-Screencast into the config are there any logs/errors in the terminal?

Can you run:
```
display=:0 npm start >> log.log
```

Allow the MM to start up, then you can kill it and paste the `log.log` file in here?

thanks!

**Comment by kr1st4ps** [comment]
> magicmirror@2.7.1 start /home/pi/MagicMirror
> sh run-start.sh

Starting MagicMirror: v2.7.1
Loading config ...
Loading module helpers ...
No helper found for module: clock.
Initializing new module helper ...
Module helper loaded: calendar
No helper found for module: currentweather.
No helper found for module: weatherforecast.
Initializing new module helper ...
Module helper loaded: MMM-NowPlayingOnSpotify
Loading module helpers ...
No helper found for module: clock.
Initializing new module helper ...
Module helper loaded: calendar
No helper found for module: currentweather.
No helper found for module: weatherforecast.
Initializing new module helper ...
Module helper loaded: MMM-NowPlayingOnSpotify
Whoops! There was an uncaught exception...
MagicMirror will not quit, but it might be a good idea to check why this happened. Maybe no internet connection?
If you think this really is an issue, please open an issue on GitHub: https://github.com/MichMich/MagicMirror/issues
Launching application.
Shutting down server...

**Comment by Tech-Smarts** [comment]
Remove the Spotify module and reinstall it.

**Comment by Tech-Smarts** [comment]
If that does not work, then you might have to completely reinstall Magic Mirror 2.

**Comment by kevinatown** [comment]
@kr1st4ps I think @Tech-Smarts was on to something here. This looks like an issue with `MMM-NowPlayingOnSpotify`, it looks your MM dies right after that gets loaded. Also I don't see any mention of MMM-Screencast.

**Comment by Tech-Smarts** [comment]
Yeah, you will just have to remove the MMM-Screencast module and remove it from the config file.  You can set it up again after you do that.  Oh and thank you Kevinatown for giving me that solution.  I am out of town so I’ll try it when I get back.  But thanks.

**Comment by maximilianfriedrich** [comment]
> Remove the Spotify module and reinstall it.

Hi,

I have the same issue. How do I remove/unintall NowPlayingOnSpotify properly? And what should I do next? Reinstall Screencast or also uninstall Screencast and reinstall?

I went to MMM-NowPlayingOnSpotiy directory in modules and typed `npm uninstall` but I don't know if it is properly removed now.

**Comment by kevinatown** [comment]
Hey @maximilianfriedrich, `npm uninstall` just uninstalls the dependencies for NowPlayingOnSpotify, if you only do this and keep the config in the `config.js` then your MM may not work. To remove NowPlayingOnSpotify from being used in your MM, simply remove it's configs from the `config.js`. 

If you don't want to keep NowPlayingOnSpotify at all then you'll need to remove the it from the `modules/` folder, however this is not required. 

I hope this helps!

**Comment by maximilianfriedrich** [comment]
Hey @kevinatown,
thank you for your kind reply and your patience and sorry for my late response.
I was busy with a photo booth and an arcade raspberry pi project. Now I finally got the time to take a look at my MagicMirror again.

I followed your suggestion and used `npm uninstall` in the `Screencast` and in the `NowPlayingOnSpotify` folder. Afterwards I remove both folders from the `modules` directory and the sections from `config.js`.

After rebooting I cloned the git of `Screencast` again into `modules` and used `npm install` in the `MMM-Screencast` directory. And finally ... BOOOM ... Screencast appears in the Youtube app and casting is possible :).

Thank you so much for your support, my issue is resolved and Screencast works perfectly! 👍 

@all get rid of the `NowPlayingOnSpotify` module. I'm using `MMM-Scrobbler`, which is linked to my LastFM account and it does basically the same thing to the mirror. The only difference is that the artwork vanishes when I pause a track on Spotify.
